exporter/importables.go

```diff
@@ -671,10 +677,6 @@ var resourcesMap map[string]importable = map[string]importable{
 					log.Printf("Skipping non-direct group %s/%s for user %s", g.Value, g.Display, username)
 					continue
 				}
-				ic.Emit(&resource{
-					Resource: "databricks_group",
-					ID:       g.Value,
-				})
```

Comment:
This should be restored back, as we may have a situation when user is discovered from thing like email notification, etc., but we also request to extract groups - without this code groups won't be exported

Reply:
Can you please elaborate how a new user can be discovered that isn't already in the path to be discovered through a group membership? 
There is only 2 references to databricks_user emission in importables.go and that's in the databricks_group Import implementation and one in the databricks_permissions' import

Reply:
user will be discovered via List only if you have `-listing users`, otherwise users will be discovered only by transitive dependencies